Packaging/RedHat/nextspace_install.sh

```diff
@@ -45,6 +45,19 @@ if [ $EPEL_REPO != "" ]; then
     fi
 fi
 
+# Drivers on real hardware
+echo -e -n "\e[1m"
+echo -n "Is it a real hardware install (drivers needed)? [yn]: "
+echo -e -n "\e[0m"
+read YN
+if [ $YN = "y" ]; then
+    echo -n "Installing complete X Window system..."
+    yum -y -q groupinstall "X Window system" 2>&1 > /dev/null
```

Comment:
`groupinstall "X Window system"` installs 174 additional packages (and use additional 300+ MB of disk space) which are not useful for NEXTSPACE. I think this line should be changed to `yum -y install xorg-x11-drivers` if you want to install additional drivers for graphics cards.

The better approach is to add `xorg-x11-drivers` dependency to package `nextspace-applications`. I want this install script as simple as possible.

Reply:
OK, but you don't need those dependencies in packages for testing on virtual machines. Before 1.0 it might not be useful.
To make life easier on real hardware, I suggest to add
- xorg-x11-drivers
- xorg-x11-xinit
- xorg-x11-utils
- xvattr

Reply:
> OK, but you don't need those dependencies in packages for testing on virtual machines. Before 1.0 it might not be useful.
>
Agree.
> To make life easier on real hardware, I suggest to add
> 
>     * xorg-x11-drivers
OK
>     * xorg-x11-xinit
Why you need this on NEXTSPACE? Xorg server start and stop is controlled by Login application. Xorg start options can be adjusted through the Login application preferences (/usr/NextSpace/Preferences/Login)...
>     * xorg-x11-utils
I agree this package contains some useful utilities for X11 development.
>     * xvattr
I have never used it. Out of curiosity: what is your use case?

Reply:
xorg-x11-xinit is very useful if something goes wrong. And you know that it allways happened. As it is basicaly allways installed, it takes me some minutes to understand why I couldn't use those utilities. In other words : user expect to have them installed. 

xvattr help to set brightness (... if something goes wrong). Well this is only for those that know about it. We can remove.

Reply:
Good. Please replace `groupinstall` with selected packages install: xorg-x11-driver, xorg-x11-init, xorg-x11-utils, xvattr.
Also note that question about real hardware is not true: `nextspace-applications` depends on `xorg-x11-drv-intel` that covers 90% of notebooks. Machines with AMD and Nvidia graphics cards often require additional setup steps (proprietary driver).

Reply:
I don't have intel graphics on my laptops and never install proprietary drivers :-) Radeon and Nouveau drivers work very well.

Reply:
About real hardware, you're right : xorg-x11-drivers depends on some drivers useful in VM, like qxl and vmware ;-))